### PR TITLE
Fix deps

### PR DESCRIPTION
**Pull request by jonashaag**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jonashaag** [comment]
@conda-forge-admin rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,20 +10,21 @@ source:
...
-    - python >=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*
-    - pip
+    - python
+    - importlib-metadata  # [py<38]
     - markupsafe >=0.9.2
+    - pip
   run:
-    - python >=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*
+    - python
+    - importlib-metadata  # [py<38]
```
Comment (xhochy) [inline recipe/meta.yaml]:
```suggestion
    - importlib-metadata
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,20 +10,21 @@ source:
...
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  number: 0
+  number: 1
 
 requirements:
   host:
-    - python >=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*
-    - pip
+    - python
+    - importlib-metadata  # [py<38]
```
Comment (xhochy) [inline recipe/meta.yaml]:
```suggestion
    - importlib-metadata
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -10,20 +10,21 @@ source:
   sha256: 9a7c7e922b87db3686210cf49d5d767033a41d4010b284e747682c92bddd8b39
 
 build:
-  noarch: python
```
Comment (xhochy) [inline recipe/meta.yaml]: Please keep this, `importlib_metadata` is an empty package on newer Python versions.

**Comment by jonashaag** [comment]
@conda-forge-admin rerender

**Pull request merged**